docs/sdk/sdk-js.md

```diff
@@ -207,6 +208,34 @@ export const HelloWidget: FC = () => {
 };
 ```
 
+### useFeatureEnabled
+This hook tells you if a lumapps feature is enabled or not: 
+
+```javascript
+import React, { FC, useMemo } from 'react';
+
+
+import { useFeatureEnabled } from 'lumapps-sdk-js';
+
+export const HelloWidget: FC = () => {
```

Comment:
It could be interesting to provide a list of feature partner can check (not all our features but some important one, I'll try to give the list)